src/services/million-verifier-service.ts

```diff
@@ -0,0 +1,28 @@
+import { AxiosInstance as axios } from '@/utils/'
+
+/**
+ * Class that provides integration with the Million Verifier API.
+ */
+export default class MillionVerifierService {
+  /**
+   * Verifies an email address in real time.
+   * @param email the email address to verify
+   * @returns whether the email address is valid
+   */
+  static async isValidEmail (email: string): Promise<boolean> {
+    const SINGLE_API_BASE_URL = 'https://api.millionverifier.com/api/v3'
```

Comment:
I don't think we need external config for this since there's only 1 endpoint (no dev or test).

However, we may want to return True if there is no API key, so that we don't burn up credits in Dev.